src/azul/chalice.py

```diff
@@ -26,9 +26,17 @@ def route(self, path, enabled=True, path_spec=None, method_spec=None, **kwargs):
 
         :param path: See https://chalice.readthedocs.io/en/latest/api.html#Chalice.route
 
-        :param method_spec: FIXME: https://github.com/DataBiosphere/azul/issues/1613
+        :param path_spec: Corresponds to an OpenAPI Paths Object. See
+                          https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#pathsObject
+                          If multiple `@app.route`s refer to the same path (but
+                          with different methods) path_spec need only be
```

Comment:
```suggestion
                          with different HTTP methods) this argument need only be
```

Avoid duplicating identifiers in documentation. Makes refactorings easier and less prone to missing references.